<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>systemadminview</class>
 <widget class="QDialog" name="systemadminview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1284</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="table_account">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>701</width>
     <height>451</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Account ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Employee ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>First Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Middle Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Last Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Username</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Lock Status</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Assistant Admin</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="Data_Pass">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>260</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Pass Data -&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>811</x>
     <y>31</y>
     <width>451</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QPushButton" name="addNewAccount">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Add new user account</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Account ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="AccountID_input">
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Employee ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="EmployeeID_input">
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>First Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="FirstName_input">
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Middle Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="MiddleName_input">
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Last Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LastName_input">
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>User Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="UserName_input">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_resetPass">
        <property name="text">
         <string>Reset Password</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QCheckBox" name="AA_checkBox">
        <property name="text">
         <string>Assistent Administrator</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="sysadmin_checkBox">
        <property name="text">
         <string>System Administrator</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Instructor_checkBox">
        <property name="text">
         <string>Instructor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="TA_checkBox">
        <property name="text">
         <string>TA</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QCheckBox" name="Administractor_checkBox">
        <property name="text">
         <string>Administractor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="block_checkBox">
        <property name="text">
         <string>Block</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QPushButton" name="Modify_Comfirm">
        <property name="text">
         <string>Modify</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Delete">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>239</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <widget class="QPushButton" name="pushButton_logout">
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Load_table">
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_ChangePass">
      <property name="text">
       <string>Change Pass</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
